src/hotspot/share/gc/g1/g1YoungGCPostEvacuateTasks.cpp

```diff
@@ -438,7 +438,7 @@ class FreeCSetStats {
   }
 
   void account_failed_region(HeapRegion* r) {
-    size_t used_words = r->marked_bytes() / HeapWordSize;
```

Comment:
At this point, for evacuation failure regions, `marked_bytes() == live_bytes()` so it can be replaced without any change.